app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      def index
+        page_size = params[:page_size]
```

Comment:
If a request is made without the parameter `:page_size`, the variable page_size will be evaluated to nil, and this can break the execution later on, resulting in a 500 status code.
It's a good practice to require mandatory parameters like so: `page_size = params.require[:page_size]`.